prow/crier/crier.go

```diff
@@ -265,7 +265,7 @@ func (s *Server) handle(r Report) error {
 			return fmt.Errorf("error creating comment: %v", err)
 		}
 	} else if len(entries) > 0 {
-		if err := s.ghc.EditComment(r.RepoOwner, r.RepoName, r.Number, createComment(r, entries)); err != nil {
```

Comment:
we're storing comments by their ID for the whole repo, not against specific issues?